docs/source/transaction_family_specifications.rst

```diff
@@ -2,7 +2,69 @@
 Transaction Family Specifications
 *********************************
 
-.. include:: transaction_family_specifications/example_transaction_families.rst
+Sawtooth includes several transaction families as examples for developing
+your own transaction family.
+
+* The :doc:`/transaction_family_specifications/blockinfo_transaction_family`
+  provides a way to store information about a configurable number of historic
+  blocks. BlockInfo is written in Python.
+  The family name is ``block_info``;
+  the associated transaction processor is ``block-info-tp``.
+
+* The :doc:`/transaction_family_specifications/identity_transaction_family`
+  is an extensible role- and policy-based system for defining permissions in a
+  way that can be used by other Sawtooth components.
+  Identity is written in Python.
+  The family name is ``sawtooth_identity``;
+  the associated transaction processor is ``identity-tp`` (see
+  :doc:`/cli/identity-tp`).
+
+* The :doc:`/transaction_family_specifications/integerkey_transaction_family`
+  simply sets, increments, and decrements the value of entries stored in a state
+  dictionary. IntegerKey is available in Go, Java, and JavaScript (Node.js).
```

Comment:
Does it make sense to also update this list to include Python and Rust at this time?

Reply:
I would rather handle changes to the "available languages" list in a separate PR, so that this PR is focused on fixing the search problem.